openquake/hazardlib/tests/gsim/kotha_2020_test.py

```diff
@@ -0,0 +1,251 @@
+# -*- coding: utf-8 -*-
+# vim: tabstop=4 shiftwidth=4 softtabstop=4
+#
+# Copyright (C) 2014-2020 GEM Foundation
+#
+# OpenQuake is free software: you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OpenQuake is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with OpenQuake. If not, see <http://www.gnu.org/licenses/>
+from openquake.hazardlib.gsim.base import CoeffsTable
+from openquake.hazardlib.gsim.kotha_2020 import (KothaEtAl2020SERA,
+                                                 KothaEtAl2020Site,
+                                                 KothaEtAl2020Slope,
+                                                 KothaEtAl2020SERASlopeGeology)
+from openquake.hazardlib.tests.gsim.utils import BaseGSIMTestCase
+
```

Comment:
Would be useful to have a comment explaining how you computed the verification tables